src/sync/sync_manager.hpp

```diff
@@ -104,7 +106,7 @@ friend class SyncSession;
     std::shared_ptr<SyncUser> get_user(const std::string& identity,
                                        std::string refresh_token,
                                        util::Optional<std::string> auth_server_url=none,
-                                       bool is_admin=false);
+                                       bool do_not_persist=false);
```

Comment:
Is it intentional that the sense of this was inverted vs what `SyncUser`'s constructor takes?